pkgs/top-level/all-packages.nix

```diff
@@ -11740,7 +11728,7 @@ with pkgs;
     boost = boost178.override { enableShared = false; };
     inherit (darwin.apple_sdk.frameworks) CoreFoundation Security;
     stdenv = if stdenv.hostPlatform.isDarwin then
-      darwin.apple_sdk_11_0.stdenv.override (old: {
+      stdenv.override (old: {
         hostPlatform = old.hostPlatform // { darwinMinVersion = "10.14"; };
         buildPlatform = old.buildPlatform // { darwinMinVersion = "10.14"; };
         targetPlatform = old.targetPlatform // { darwinMinVersion = "10.14"; };
```

Comment:
All this stuff is unnecessary now.

Reply:
I'll also check if I can clean up the mongodb expressions in another PR.

Reply:
Those aren’t built on Hydra FWIW, so you can do whatever.